main.tf

```diff
@@ -92,11 +97,13 @@ resource "google_storage_bucket" "bucket" {
 }
 
 resource "google_filestore_instance" "nfs" {
-  count = var.filestore_disabled ? 0 : 1
+  count    = var.filestore_disabled ? 0 : 1
+  provider = google
 
-  name = local.uuid
-  tier = "STANDARD"
-  zone = local.zone
+  name     = substr(local.uuid, 0, 53)
```

Comment:
why do we shorten it? if its 53 here and 55 above, why not just chop it off at 50 or whatever above.
we shouldn't really call it a uuid since there is none in there and its not guaranteed to be unique given its a name + some random characters which we keep shortening.